Onboarding/onboarding-part-2.md

```diff
@@ -376,72 +396,167 @@ card EDDConfirmation, then: SaveEDDAndContinue do
 
   page =
     get(
-      "https://platform-mnch-contentrepo.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
       headers: [
         ["Authorization", "Token @config.items.contentrepo_token"]
       ],
       query: [["whatsapp", "true"]]
     )
 
   edd_date_full = date(edd_date_year, edd_date_month, edd_day)
-  edd_date_full_str = datevalue(edd_date_full, "%Y-%m-%d")
-  question = substitute("@page.body.body.text.value.message", "{{edd}}", "@edd_date_full_str")
+  month_name = datevalue(edd_date_full, "%B")
+  question = substitute("@page.body.body.text.value.message", "{dd}", "@edd_day")
+  question = substitute("@question", "{month_name}", "@month_name")
+  question = substitute("@question", "{yyyy}", "@edd_date_year")
+
+  message = page.body.body.text.value
+  button_labels = map(message.buttons, & &1.value.title)
 
   buttons(
-    SaveEDDAndContinue: "Yes it is correct",
-    EDDMonth: "Change date"
+    SaveEDDAndContinue: "@button_labels[0]",
+    PregnantEDDMonth: "@button_labels[1]"
   ) do
     text("@question")
   end
 end
 
 ```
 
-# SaveEDDAndContinue
-
-Sends the user to the next part of the onboarding flow.
+## SaveEDDAndContinue
 
 ```stack
-card SaveEDDAndContinue do
-  update_contact(onboarding_part_2: "complete")
-  update_contact(edd: "@edd_date_full_str")
+card SaveEDDAndContinue, then: ContinueEDDBranch do
+  edd_date_full_str = datevalue(edd_date_full, "%Y/%m/%d")
```

Comment:
Is there a reason that we're using this date format? Could we rather use the `full-date` format of [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6)?

Reply:
100% I'm using this because I copied this code from what existed before. Line 357 I think uses this date format.